src/peung/peung.py

```diff
@@ -3,10 +3,9 @@
 from time import sleep
 from pathlib import Path
 from platformdirs import user_data_dir
+from gtts import gTTS
```

Comment:
need to add `"gtts"` to [here](https://github.com/yusufmte/peung/blob/c00c2f1b0f9c64d970eac1431f75c69bde915060/pyproject.toml#L20-L24) for this to get installed.

right now if you create a new python environment and try to install and run peung it will complain at this line

Reply:
ahh, peup, ty!!!